src/golang/lib/job/databricks.go

```diff
@@ -0,0 +1,180 @@
+package job
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	databricks_lib "github.com/aqueducthq/aqueduct/lib/databricks"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/dropbox/godropbox/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	databricksFunctionScript = "aqscript.py"
+	databricksParamScript    = "paramScript.py"
+	databricksMetricScript   = "metricScript.py"
+	databricksDataScript     = "dataScript.py"
+)
+
+type DatabricksJobManager struct {
+	databricksClient *databricks.WorkspaceClient
+	conf             *DatabricksJobManagerConfig
+	runMap           map[string]int64
```

Comment:
I just realized this - the `process` job manager uses a concurrent map. Do we need a concurrent map here as well?

Reply:
If we do, I'm fine with doing it in another PR. This PR is already large and we won't release the whole feature anyway